polymorphic_value.h

```diff
@@ -73,7 +73,7 @@ template<typename T, polymorphic_value_options Options = polymorphic_value_optio
     polymorphic_value() {}
     polymorphic_value(nullopt_t) {}
     polymorphic_value(const polymorphic_value& src) requires copyable {
-        src.m_handler.copy(*this, src.m_data);
+        src.getHandlerBase().copy(*this, src.m_data);
```

Comment:
Once the first issue is resolved then this is the next issue I hit, similar in nature to the previous one.

Reply:
Yes, I think it should be replaced everywhere (more or less) that m_handler is used.

Are you guys working on forks of my repo? I just fetched and I don't see any more branches than main.

Reply:
That was my suspicion.  We should ensure all paths are tested.  I can add code coverage if you like?

Yes, we are both on forks (which makes it easier when setting up github action, code coverage, etc to ensure they work correctly before merging to the main repo)

Reply:
Well I didn't intend for this little test program to be complete, I just wanted to show the principles. I'm  sure there are untested paths and there may well be bugs, but I think it works well enough to show the ideas, and how it differs from your  original implementation.

I have omitted quite a few things like noexcept etc. that can be added on if this proceeds towards standarization.